Source/WebKit/WebProcess/Storage/WebSWContextManagerConnection.cpp

```diff
@@ -592,4 +603,27 @@ void WebSWContextManagerConnection::removeNavigationFetch(WebCore::SWServerConne
     });
 }
 
+#if ENABLE(REMOTE_INSPECTOR) && PLATFORM(COCOA)
+void WebSWContextManagerConnection::connectToInspector(WebCore::ServiceWorkerIdentifier serviceWorkerIdentifier)
+{
+    RefPtr channel = ServiceWorkerDebuggableFrontendChannel::create(serviceWorkerIdentifier);
```

Comment:
Seems like this returns a Ref, not a RefPtr. You can use `channel.copyRef()` on the next line if that's what you were trying to address.